lib/fluent/plugin/out_http.rb

```diff
@@ -121,6 +127,36 @@ def configure(conf)
         end
         define_singleton_method(:format, method(:format_json_array))
       end
+
+      if @auth and @auth.method == :aws_sigv4
+        begin
+          require 'aws-sigv4'
+          require 'aws-sdk-core'
+        rescue LoadError
+          raise Fluent::ConfigError, "The aws-sigv4 and aws-sdk-core gems are required for aws_sigv4 auth. Run: 'gem install aws-sigv4 -v 1.8.0' and 'gem install aws-sdk-core -v 3.191'."
```

Comment:
Sorry, one more thing.
Does it really need to stick to aws-sigv4  v1.8.0?
aws-sdk-core v3.191 depends on `aws-sigv4 ~> 1.8`, so that latest version of aws-sigv4 1.8 series will be installed automatically when user installs aws-sdk-core v3.191 series.
It would be happy if we can omit specifying strict version of aws-sigv4 to receive latest bug fixes (especially vulnerability fixes).

Reply:
@ashie , Correct, it doesn't need to be v1.8.0 strictly. I have updated the code in the last commit.

The error message now looks like:

```
2024-04-05 08:46:10 -0500 [error]: config error file="fluent.conf" error_class=Fluent::ConfigError error="The aws-sdk-core and aws-sigv4 gems are required for aws_sigv4 auth. Run: gem install aws-sdk-core -v '~> 3.191'"
```

Reply:
Thanks!